aptos-move/framework/aptos-framework/sources/coin.move

```diff
@@ -138,6 +152,22 @@ module aptos_framework::coin {
     /// Capability required to burn coins.
     struct BurnCapability<phantom CoinType> has copy, store {}
 
+    struct FungibleAssetMigration has key {
+        // Fast lookup maps to help convert back and forth between coins and fungible assets.
+        coins_to_fungible_assets: SmartTable<TypeInfo, Object<Metadata>>,
+        fungible_assets_to_coins: SmartTable<Object<Metadata>, TypeInfo>,
+
+        // If set to true, coin::transfer and coin::deposit will always deposit the amount
+        // as fungible assets instead of Coins.
+        enable_default_fungible_assets_in_transfers: bool,
+    }
+
+    struct FungibleAssetAdmin<phantom CoinType> has key {
```

Comment:
nit: better to include "Migration" in the name to indicate this struct deals with v1->v2 migration

Reply:
could you reuse `Object<metadata>` and put this into that object within the same resource_group?

Reply:
@lightmark i added `#[resource_group_member(group = aptos_framework::object::ObjectGroup)]` - is that all i need to do?

Reply:
Yeah